lib/ReactViews/Search/SearchBox.jsx

```diff
@@ -38,13 +42,10 @@ export default React.createClass({
     },
 
     searchWithDebounce() {
```

Comment:
You should be able to remove this function entirely, and instead do `this.searchWithDebounce = debounce(this.search, 2000);` in `componentWillMount`.